woocommerce/woocommerce-android
# Feature/bulk update stock quantity

This PR adds bulk updating of stock quantity to product variations.

Note that the bulk updating only applies to variations that are stock managed. The alternative was to change the stock managed value to `true` for all variations, which seemed like the wrong thing to do.

We also had a Slack conversation ( p1667399352822669-slack-CGPNUU63E ) involving re-designing the approach to bulk updating, because right now it expects a separate fragment and view model for every field that supports bulk updating. As more fields get supported, this would be unmanageable.

I think that a base bulk update fragment which handles the Done button, snackbars, progress dialogs, etc., would be a good start to resolving this. I considered doing just that but given that Friday is my last day on the team I didn't think I'd have enough time, so we stuck with the existing approach. I tried to mimic the way bulk update of prices works as closely as possible.

TEST A

* Tap to view product detail for a variable product that has at least one variation that's stock managed
* Tap Variations
* Tap the options menu and choose "Bulk update"
* Tap "Stock quantity"
* Enter a new value and tap Done
* Verify the stock managed variations have the updated stock quantity

TEST B

* Select "Bulk update" for a variable product that has no stock managed variations
* Ensure "Stock quantity" does not appear
